TAF_EasyRest/TestFramework/Pages/AdministratorPanelPage.cs

```diff
@@ -0,0 +1,34 @@
+﻿using TestFramework.PageComponents.AdministratorPanelComponents;
+using TestFramework.PageComponents.NavigationMenuComponents;
+
+namespace TestFramework.Pages
+{
+    public class AdministratorPanelPage : BasePage
+    {
+        NavigationMenuPageComponent NavigationMenuPageComponent { get; }
+        UserMenuHeaderButtonPageComponent UserMenuHeaderButton { get; }
```

Comment:
please specify the access modifier explicitly

Reply:
added